web/js/utils/directoryTree.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const directoryTreeId = '#directoryTree';
+
+function clearSelectedNode() {
+    const directoryTree = $(directoryTreeId);
+
+    const state = directoryTree.tree('getState');
+    state.selected_node = [];
+    directoryTree.tree('setState', state);
+}
+
+function createNodeId(type, name) {
+    return `jqtree-element__${type}__${name}`;
```

Comment:
One thing to test here is creating two projects in different folders with the same base project name.  Would they both get the same id?

Reply:
Good point, there will be collisions, unless we tuck on some unique value. Will something like this be acceptable?

```js
jqtree-element__${type}__${name}__${Math.floor(Math.random() * 1000000)}
```
Or we need to pull in a helper for generating unique IDs?

Reply:
You are right, a simple counter should be sufficient here - I was overthinking it.